dev/packages/beats/couchbase-0.0.1/dataset/node/fields/fields.yml

```diff
@@ -0,0 +1,114 @@
+- key: couchbase
```

Comment:
The fields.yml in the packages don't have the global keys prefix: https://github.com/elastic/package-registry/blob/master/dev/packages/example/nginx-1.2.0/dataset/access/fields/fields.yml It is not needed anymore. This was use historically for creating configs / docs and settings. But this moved to the manifest.yml

Reply:
Thanks for the explanation. I think we can get rid of it then.

Reply:
I'll address this comment in https://github.com/elastic/package-registry/pull/240

Reply:
I understood that you referred to the "key" key only, not all keys :D Sorry for the confusion.

Reply:
Should have been more clear, sorry :-(